applications/nrf_desktop/src/modules/Kconfig.dvfs_state_template

```
@@ -0,0 +1,35 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig DESKTOP_DVFS_STATE_$(dvfs_state)_ENABLE
+	bool "Handle the DVFS $(dvfs_state) state"
+	default y
+	# depends on $(dvfs_dependencies)
+
+if DESKTOP_DVFS_STATE_$(dvfs_state)_ENABLE
+
+config DESKTOP_DVFS_STATE_$(dvfs_state)_TIMEOUT_MS
+	int "Timeout in milliseconds for the DVFS $(dvfs_state) state" if $(dvfs_timeout) != 0
+	default $(dvfs_timeout)
+	range 0 65535
+	help
+	  Timeout in milliseconds specifying time after which DVFS request related to
+	  DESKTOP_DVFS_STATE_$(dvfs_state) times out.
+	  A request is automatically cleared on timeout.
+
+choice "DESKTOP_DVFS_STATE_$(dvfs_state)_ACTIVE_FREQ"
+	prompt "Default active frequency of DESKTOP_DVFS_STATE_$(dvfs_state)"
```

for consistency with other prompts

```suggestion
	prompt "Active frequency of the DVFS $(dvfs_state) state"
```